packages/vm/examples/helpers/blockchain-mock-data.ts

```diff
@@ -0,0 +1,96 @@
+const blockchainMockData = {
```

Comment:
What does this example do? It only defines a constant?

Reply:
Meant to be exported, addressed it in latest commit